doc/database/database_schemata.md

```diff
@@ -67,25 +67,23 @@ plants {
   INT height
 }
 
-plant_detail{}
-
 plants }o--|| plant_detail: "type"
 
 plants ||--o{ seeds : ""
 
-plant_detail }o--o{ plant_detail: "likes"
+plants }o--o{ plants: "likes"
 
-plant_detail }o--o{ plant_detail: "dislikes"
+plants }o--o{ plants: "dislikes"
 
 species{}
-genus{}
-subfamily{}
-family{}
+genera{}
+subfamilies{}
+families{}
 
-plant_detail }|--|| species : ""
-species }|--|| genus : ""
-genus }|--|| subfamily : ""
-subfamily }|--|| family : ""
+plants }|--|| species : ""
+species }|--|| genera : ""
+genera }|--|| subfamilies : ""
+subfamilies }|--|| families : ""
```

Comment:
Same as with the other comment.
Maybe we can also leave it as it is and I'll update the schema with the PR that implements hierarchical information.

Reply:
Yes I will leave it as is. I saw that the master branch has some code examples in the database_schemata that makes use of the tables you suggested to remove.